### PR TITLE
InfernalVoidScans: Update domain

### LINKED ISSUE #5377: Infernal Void Scans (HiveScans) URL Change (.net > .com)
### Source information

Infernal Void Scans 1.4.39

### Source language

English

### Source new URL

hivetoon.com

### Other details

_No response_

### Acknowledgements

- [X] I have updated all installed extensions.
- [X] I have opened WebView and checked that the source URL has not changed yet.
- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

### PR DESCRIPTION
**Pull request by choppeh**
Closes #5377

Checklist:

- [ ] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [x] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [x] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [x] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [x] Have not changed source names
- [ ] Have explicitly kept the `id` if a source's name or language were changed
- [x] Have tested the modifications by compiling and running the extension through Android Studio
- [ ] Have removed `web_hi_res_512.png` when adding a new extension

### CODE REVIEW

File: src/en/infernalvoidscans/build.gradle [hunk 1]
```diff
@@ -2,8 +2,9 @@ ext {
     extName = 'Infernal Void Scans'
     extClass = '.InfernalVoidScans'
     themePkg = 'mangathemesia'
-    baseUrl = 'https://hivetoon.net'
-    overrideVersionCode = 9
+    baseUrl = 'https://hivetoon.com'
+    overrideVersionCode = 10
+    isNsfw = true
```
Comment (vetleledaal) [inline src/en/infernalvoidscans/build.gradle]: What series specifically makes it NSFW?

Comment (choppeh) [reply src/en/infernalvoidscans/build.gradle]: > What series specifically makes it NSFW?

- [The Best Noble in Another World: The Bigger My Harem Gets, the Stronger I Become](https://hivetoon.com/manga/the-best-noble-in-another-world-the-bigger-my-harem-gets-the-stronger-i-become/) - Chapter 2
- [Brutal: Confessions of a Homicide Investigator](https://hivetoon.com/manga/brutal-confessions-of-a-homicide-investigator/) - Chapter 3

```Adult``` option available in the filters

Comment (Riztard) [reply src/en/infernalvoidscans/build.gradle]: but generally looks safe. not sure since we got no thing in between safe and nsfw

Comment (AwkwardPeak7) [reply src/en/infernalvoidscans/build.gradle]: I agree, it isn't really nsfw as a whole

**Review by AwkwardPeak7 — APPROVED** [review]

**Pull request merged**